package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
+    "build": "skyux build-public-library",
+    "start": "skyux serve"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/animations": ">=7.2.2",
+    "@angular/common": ">=7.2.2",
+    "@angular/compiler": ">=7.2.2",
```

Comment:
Do you want this library to be used by both SKY UX 2 and SKY UX 3 SPAs? If so, you should probably mimic the peer dependency structure of our component libraries (i.e., all Angular dependencies start at 4).
See: https://github.com/blackbaud/skyux-core/blob/master/package.json#L26-L28